doc/developers/contributing.rst

```diff
@@ -261,8 +261,13 @@ modifying code and submitting a PR:
 
 .. note::
 
-  If you are modifying a Cython module, you have to re-run step 5 after modifications
-  and before testing them.
+    If you are modifying a Cython module, you have to re-compile after
+    modifications and before testing them::
+
+        pip install --no-build-isolation -e .
```

Comment:
I think in practice many of us do `make in` instead though. The `pip install -e` thingy is needed only once, I think.

Reply:
Right, but we don't document `make in` as the main way to go because it's not on all platform and not used in all projects (numpy does not have a Makefile for instance).